spiel/ut/spiel_ut_common.h

```diff
@@ -33,7 +33,7 @@
 
 #define SERVER_ENDPOINT_ADDR "0@lo:12345:34:1"
 #define SERVER_ENDPOINT      M0_NET_XPRT_PREFIX_DEFAULT":"SERVER_ENDPOINT_ADDR
-#define CLIENT_ENDPOINT_ADDR "0@lo:12345:34:*"
+#define CLIENT_ENDPOINT_ADDR "0@lo:12345:33:*"
```

Comment:
Perhaps #include "conf/ut/common.h" instead of duplicating?

Reply:
Done.